lib/node_modules/@stdlib/blas/ext/base/scusumkbn/manifest.json

```
@@ -1,40 +1,86 @@
 {
-	"options": {},
-	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
-	],
-	"confs": [
-		{
-			"src": [
-				"./src/scusumkbn.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": []
-		}
-	]
-}
+  "options": {},
+  "fields": [
+	{
+	  "field": "src",
+	  "resolve": true,
+	  "relative": true	
+	},
+	{
+	  "field": "include",
+	  "resolve": true,
+	  "relative": true	
+	},
+	{
+	  "field": "libraries",
+	  "resolve": false,
+	  "relative": false	
+	},
+	{
+	  "field": "libpath",
+	  "resolve": true,
+	  "relative": false	
+	}
+  ],
+  "confs": [
+	{
+	  "task": "build",
+	  "src": [
+	    "./src/scusumkbn.c"
+	  ],
+	  "include": [
+		"./include"
+	  ],
+	  "libraries": [
+		"-lm"
+	  ],
+	  "libpath": [],
+	  "dependencies": [
+	      "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-float",
+        "@stdlib/napi/argv-int64",
+        "@stdlib/napi/argv-strided-float32array"
+	  ]
+	},
+	{
+	  "task": "benchmark",
+	  "src": [
+		"./src/scusumkbn.c"
+	  ],
+	  "include": [
+		"./include"
+	  ],
+	  "libraries": [
+		"-lm"
+	  ],
+	  "dependencies": [
+		"@stdlib/napi/export",
+		"@stdlib/napi/argv",
+		"@stdlib/napi/argv-float",
+		"@stdlib/napi/argv-int64",
+		"@stdlib/napi/argv-strided-float32array"
+	  ]
+	},
+	{
+	  "task": "examples",
+	  "src": [
+		"./src/scusumkbn.c"
+	  ],
+	  "include": [
+		"./include"
+	  ],
+	  "libraries": [
+		"-lm"
+	  ],
+	  "libpath": [],
+	  "dependencies": [
+		"@stdlib/napi/export",
+		"@stdlib/napi/argv",
+		"@stdlib/napi/argv-float",
+		"@stdlib/napi/argv-int64",
+		"@stdlib/napi/argv-strided-float32array"
+	  ]
+	}
+ ]
+}
```

```suggestion
}

```

Missing trailing newline. Have you setup EditorConfig?